### DISCUSSION TITLE
如何客製化select的check icon

### DISCUSSION THREAD

**Discussion by janlin002** [ant-design/ant-design#42019] [Q&A] [answered]
antd 的 select 預設勾選是在右邊，該如何寫才能換到右邊，並且改成checkbox(下圖是我希望的效果)
![截圖 2023-04-26 下午9 43 04](https://user-images.githubusercontent.com/68535430/234595055-19a1eeaf-9c07-49b6-94a6-d27ee944e6ab.png)
![截圖 2023-04-26 下午9 43 17](https://user-images.githubusercontent.com/68535430/234595084-17625501-b7a3-47a0-896c-8f99e0aa591f.png)

**Comment by zombieJ** [comment, marked as answer]
这个看起来可以用一层的 TreeSelect：
https://ant.design/components/tree-select-cn#components-tree-select-demo-checkable

**Reply by janlin002** [reply]
感謝你，學到了
